lms/djangoapps/bulk_email/toggles.py

```diff
@@ -0,0 +1,19 @@
+"""
+Toggles for bulk_email app
+"""
+
+from django.conf import settings
+
+
+# .. toggle_name: bulk_email.EMAIL_USE_DEFAULT_FROM_FOR_BULK
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: False
+# .. toggle_description: If True, use the same DEFAULT_FROM_EMAIL as the from_addr for all bulk email, to avoid issues with spam filtering
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2020-10-01
+# .. toggle_tickets: OSPR-4957
+
+
+def is_email_use_default_from_bulk_enabled():
+    # TODO: Replace function with SettingToggle when it is available.
+    return getattr(settings, "EMAIL_USE_DEFAULT_FROM_FOR_BULK", False)
```

Comment:
I believe SettingToggle should now be available, right?

Reply:
any reference to it then ?

Reply:
Until we have more permanent docs, you can search for "SettingToggle" in this doc: https://openedx.atlassian.net/wiki/spaces/AC/pages/2248409171/Toggles+and+Settings+Doc-a-thon+2021